backend/src/main/resources/application.yml

```
@@ -62,3 +65,7 @@ management:
   endpoint:
     prometheus:
       enabled: true
+
+apply:
+  post:
+    base-url: ${APPLY_POST_BASE_URL}
```

depth를 3개까지 둔 이유가 있을까요?

```suggestion
base-url: 
    applyform:
        ${APPLY_POST_BASE_URL}
```

---

기존 코드 베이스에서 해당 용어를 사용하고 있어서 그대로 사용했습니다.